cves/kernel/CVE-2021-3491.yml

```diff
@@ -389,37 +387,38 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: This vulnerability is only an issue with input that is outside of
+          the expected input.
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
+    applies: false
     note:
   yagni:
-    applies:
+    applies: false
     note:
   complex_inputs:
-    applies:
+    applies: false
```

Comment:
Do you think this vulnerability could fall under this category too?

Reply:
I though that too, but then I looked up the Project's definition of complex inputs and I was less sure it applied
[Link](https://vulnerabilityhistory.org/articles/beware-complex-inputs)